src/less/chip/chip.less

```diff
@@ -0,0 +1,60 @@
+@import "../variables/variables.less";
+
+.chip {
+    align-items: center;
+    background-color: var(--color-background-tertiary);
+    border-radius: @spacing-400;
+    box-sizing: border-box;
+    color: var(--color-foreground-primary);
+    display: inline-flex;
+    font-size: @font-size-14;
+    gap: 2px;
+    height: @spacing-400;
+    justify-content: center;
+    max-width: 320px;
+    min-width: 56px;
+    padding: @spacing-75 @spacing-150;
+}
+
+.chip__text {
+    margin-top: 2px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+button.chip__button {
+    background-color: transparent;
+    border: 2px solid transparent;
+    border-radius: 50px;
+    box-sizing: border-box;
+    flex-shrink: 0;
+    height: @spacing-300;
+    line-height: 0;
+    margin-right: -6px;
```

Comment:
Hmm, this seems arbitrary, why what is adding the margin when button is there?

Reply:
This ensures the component looks great with and without a close icon without needing additional tweaks or modifiers.